pimd/pim_cmd.c

```diff
@@ -7374,12 +7374,12 @@ DEFUN (ip_pim_packets,
 
 DEFUN (no_ip_pim_packets,
        no_ip_pim_packets_cmd,
-       "no ip pim packets (1-100)",
+       "no ip pim packets [(1-65535)]",
        NO_STR
        IP_STR
        "pim multicast routing\n"
        "packets to process at one time per fd\n"
-       "Number of packets\n")
+       IGNORED_IN_NO_STR)
 {
 	char default_packet[3];
```

Comment:
Same here: either stringnify define or use `DESTROY` in the YANG op.

Reply:
changed to destroy